integration_tests/integration_tests/tests/conftest.py

```diff
@@ -26,6 +32,14 @@ def init_tests_env(target, tmp_path_factory, worker_id: str):
             env_ready_indicator_path.touch()
 
 
+@pytest.fixture(autouse=True)
+def skip_by_targets(request, target):
+    if request.node.get_closest_marker("skip_targets"):
+        skipped_targets = request.node.get_closest_marker("skip_targets").args
```

Comment:
From this code I understand that `skip_targets` takes a list, but it seems that you pass `"databricks"`.
What am I missing?

Reply:
I made it accept *args but maybe it's confusing
I'll change to a list